src/preact/services/storage.js

```diff
@@ -0,0 +1,33 @@
+class StorageService {
+  /** */
+  constructor() {
+    // eslint-disable-next-line local/no-forbidden-terms
+    this.storage_ = self.localStorage;
+  }
+
+  /**
+   * @param {string} key
+   * @return {Promise<any|undefined>}
+   */
+  async get(key) {
```

Comment:
This is marked async, but there's no async behavior. Did you forget an await somewhere?

Reply:
Was initially trying to port the existing `storage-impl` service, which had an async signature.
But this file has been removed now anyway.